Elf/__init__.py

```diff
@@ -453,11 +455,12 @@ def _parseSectionSymbols(self):
                 symtab = self.readAtOffset(sec.sh_offset, sec.sh_size)
 
                 count, remain = divmod(sec.sh_size, len(sym))
-                syms = sym * count
 
-                vstruct.VArray(elems=syms).vsParse(symtab, fast=True)
+                syms = vstruct.VArray()
+                [syms.vsAddElement(self._cls_symbol(bigend=self.bigend)) for i in range(count)]
```

Comment:
you no longer use `sym`, defined above as `self._cls_symbol(bigend=self.bigend)`